main.go

```diff
@@ -72,6 +73,26 @@ func runCommand() error {
 		return nil
 	}
 
+	// read tsa and include-certs from gitconfig
+	path, err := os.Getwd()
+	if err == nil {
```

Comment:
Is there a reason we want to "fail open" here and not return if an error is returned?

Reply:
read tsa and include-cert should be optional, so just use the defaults if these are not defined within git config